packages/core/build/addresses/mainnet.json

```diff
@@ -34,34 +34,34 @@
     },
```

Comment:
this file shouldn't be removed because [defillama](https://github.com/DefiLlama/DefiLlama-Adapters/blob/c091de607442c1430b00770c77ad2f737d6e0ed4/projects/hop/index.js#L18) relies on it (sorry it wasn't clear)

Reply:
yes